main.go

```diff
@@ -64,8 +62,13 @@ func buildRegistryCommand(cmdName string, registry Registry, servicesConfig Serv
 		Short: "Build, tag and push an image",
 		Run: func(cmd *cobra.Command, args []string) {
 			authOption := getRegistryAuthOption(registry)
+			maxGoRoutines, err := cmd.Flags().GetInt("max-go-routines")
+			if err != nil {
+				finishWithError("failed getting max-go-routines flag", err)
```

Comment:
Instead of always calling this function we can use `RunE` to return an error - and handle it in a single place